mappings/net/minecraft/world/biome/BiomeGeneratorSettings.mapping

```diff
@@ -0,0 +1,158 @@
+CLASS none/aun net/minecraft/world/biome/BiomeGeneratorSettings
+	CLASS none/aun$a
+		FIELD A useRavines Z
+		FIELD B useWaterLakes Z
+		FIELD C waterLakeChance I
+		FIELD D useLavaLakes Z
+		FIELD E lavaLakeChance I
+		FIELD F useLavaOceans Z
+		FIELD G fixedBiome I
+		FIELD H biomeSize I
+		FIELD I riverSize I
+		FIELD J dirtSize I
+		FIELD K dirtCount I
+		FIELD L dirtMinHeight I
```

Comment:
Height? Is that the vertical size or the Y position it spawns in the world? If the latter, maybe it should be `Y` or `YLevel` instead?